deployment/terraform/batch-container-service.tf

```diff
@@ -0,0 +1,95 @@
+#
+# Security group resources
+#
+resource "aws_security_group" "batch_container_instance" {
+  vpc_id = "${module.vpc.id}"
+
+  tags {
+    Name        = "sgBatchContainerInstance"
+    Project     = "${var.project}"
+    Environment = "${var.environment}"
+  }
+}
+
+#
+# Autoscaling Resources
+#
+data "template_file" "batch_container_instance_cloud_config" {
+  template = "${file("cloud-config/ecs-batch-user-data.yml")}"
+
+  vars {
+    ecs_cluster_name = "${aws_ecs_cluster.batch_container_instance.name}"
+    environment      = "${var.environment}"
+  }
+}
+
+resource "aws_launch_configuration" "batch_container_instance" {
+  lifecycle {
+    create_before_destroy = true
+  }
+
+  ebs_optimized = true
+  # TODO: Use a separate container instance profile for Batch cluster?
+  iam_instance_profile = "${aws_iam_instance_profile.container_instance.name}"
+  image_id             = "${var.ecs_instance_ami_id}"
+  instance_type        = "${var.batch_container_instance_type}"
+  key_name             = "${var.aws_key_name}"
+  # TODO: Use a separate security group for Batch cluster instances?
+  security_groups      = ["${aws_security_group.container_instance.id}"]
+
+  user_data = "${data.template_file.batch_container_instance_cloud_config.rendered}"
+
+  # Not necessary for NVMe SSD instance stores
```

Comment:
I can delete this before merging, I just wanted to make it explicit here that this isn't necessary, since AWS automatically maps NVMe SSD ephemeral devices.